manifests/compute.pp

```diff
@@ -51,10 +52,12 @@
     enable => $felix_enable,
   }
 
-  Class['neutron'] ->
-  Package[$calico::compute_package] ->
-  File[$calico::felix_conf] ~>
-  Service[$calico::felix_service]
+  if $openstack_computehost {
```

Comment:
```if $calico::compute``` will perhaps work here? No need for a new param as we can reuse the one from the main class I think.